src/server/page_server.cpp

```diff
@@ -30,13 +30,24 @@
 #include "page_buffer.h"
 #include "server_type.hpp"
 #include "system_parameter.h"
+#include "thread_entry_task.hpp"
 #include "vpid_utilities.hpp"
 
 #include <cassert>
 #include <cstring>
 #include <functional>
 #include <thread>
 
+page_server::page_server (const char *db_name)
+  : m_server_name { db_name }
+{
+  const auto thread_count = 4; // Arbitrary chosen
+  const auto task_max_count = thread_count * 4;
```

Comment:
I don't understand why we're multiplying thread_count by 4 
Could you explain it to me ?